apps/reference/README.md

```diff
@@ -42,4 +42,4 @@ We use documentation specifications which can be used to generate human-readable
 - CLISpec (custom to Supabase): for CLI commands and usage.
 
 The benefit of using custom specifications is that we can generate many other types from a strict schema (eg, HTML and manpages).
-It also means that we can switch any documentation system we want. On this site we use Docusaurus, but in Supabase's official website we use a custom React site and expose only a subset of the available API for each tool.
+It also means that we can switch any documentation system we want. On this site we use Next.JS, but in Supabase's official website we use a custom React site and expose only a subset of the available API for each tool.
```

Comment:
just to sanity check - this is in the `reference` folder though?
we probably need the same README in `temp-docs`

Reply:
right. Copied this file in the new docs readme